Dockerfile-ngc-tensorflow-hpc

```diff
@@ -0,0 +1,28 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# Copy various shell scripts that group dependencies for install
+# Then, make sure that these scripts have execute permissions
+COPY dockerfile_scripts /tmp/det_dockerfile_scripts
+RUN chmod -R 777 /tmp
+
+ARG AWS_PLUGIN_INSTALL_DIR=/container/aws
+ARG WITH_AWS_TRACE
+ARG INTERNAL_AWS_DS
+ARG INTERNAL_AWS_PATH
+RUN if [ "$WITH_OFI" = "1" ]; then /tmp/det_dockerfile_scripts/build_aws.sh "$WITH_OFI" "$WITH_AWS_TRACE"; fi
+
+#USING OFI
+ARG AWS_LIB_DIR=${AWS_PLUGIN_INSTALL_DIR}/lib
+ENV LD_LIBRARY_PATH=${WITH_OFI:+$AWS_LIB_DIR:}$LD_LIBRARY_PATH
+
+RUN /tmp/det_dockerfile_scripts/install_libnss_determined.sh
+
+# Set an entrypoint that can scrape up the host libfabric.so and then
+# run the user command. This is intended to enable performant execution
+# on non-IB systems that have a proprietary libfabric.
+RUN mkdir -p /container/bin && \
+    cp /tmp/det_dockerfile_scripts/scrape_libs.sh /container/bin
+ENTRYPOINT ["/container/bin/scrape_libs.sh"]
+
+RUN rm -r /tmp/*
```

Comment:
can you configure your editor to always include a terminating newline?  This is an old unix standard for files, and its easier if everybody follows it.